vignettes/HouseHoldCoanalysis.Rmd

```diff
@@ -0,0 +1,248 @@
+---
+title: "Household Co-Analysis"
+output:
+  html_document:
+    toc: true
+    number_sections: true
+    toc_depth: 3
+    toc_float: true
+urlcolor: blue
+vignette: >
+    %\VignetteIndexEntry{Household Co-Analysis}
+    \usepackage[utf8]{inputenc}
+    %\VignetteEngine{knitr::rmarkdown}
+---
+
+
+```{r, echo=FALSE, out.width = "60%", out.extra='style="border: 0; padding:20px"'}
+knitr::include_graphics("GGIR-MASTERLOGO-RGB.png")
+```
+
+See also complementary vignettes on:
+[General introduction to GGIR](https://cran.r-project.org/package=GGIR/vignettes/GGIR.html), [Day segment analyses](https://CRAN.R-project.org/package=GGIR/vignettes/TutorialDaySegmentAnalyses.html), [GGIR parameters](https://CRAN.R-project.org/package=GGIR/vignettes/GGIRParameters.html), [Embedding external functions (pdf)](https://CRAN.R-project.org/package=GGIR/vignettes/ExternalFunction.pdf), and [Reading ad-hoc csv file formats](https://CRAN.R-project.org/package=GGIR/vignettes/readmyacccsv.html).
+
+
+Household co-analysis facilitates the analysis of how sleep and physical activity relate
+between members of the same household.
+
+# Accelerometer file names
+
+The household co-analysis requires that households and family member can be recognised.
+To do this we assume the following logic in the file names:
+
+`StudyNumber-HouseholdID-MemberID_anyotherinformation.bin`
+
+This example is for a .bin file, but the same applies to .cwa or .csv files.
+
+For example the files:
+
+- `001-002-001_12345-2023.bin`
+- `001-002-002_23456-2023.bin`
+- `001-002-M_23456-2023.bin`
+
+will be recognised as household ID `002` with member IDs `001`, `002`, and `M`.
+
+# Configuring GGIR
+
+The following input arguments are needed to run the Household co-analysis:
+
+- The Household co-analysis is integrated in GGIR as part 6, so we need to run GGIR with
+part 1 to 6, with `mode = 1:6`.
+- `windowsizes = c(5, 60, 3600)`. Setting the second value to 60 ensures that light and 
+temperature, if available, are aggregated per minute.
+- `part5_agg2_60seconds = TRUE`. This ensures that GGIR part 5 stores
+the time series at 1 minute resolution.
+- `part6HCA = TRUE` to tell GGIR to perform Household Co-Analysis.
+- `part6_threshold_combi = "30_100_400"` where 30, 100 and 400 need to correspond
+to the accelerometer threshold combination that is used in part 5 that you want to use
+in part 6. GGIR part 5 facilitates multiple threshold combinations but in part 6 you
+need to select one.
+- `timewindow = WW` to run part 5 analysis based on waking-up to waking-up windows.
```

Comment:
It would be helpful to explain why the household co-analysis can only be derived with timewindow = "WW". I guess the reason is that many of the variables compare wakeup time of the house members and it's easier to do that in a "WW" timewindow. Although it would also be possible in a "OO" timewindow.